financial-services-accelerator/internal-webapps/org.wso2.financial.services.accelerator.consent.mgt.endpoint/src/main/java/org/wso2/financial/services/accelerator/consent/mgt/endpoint/api/ConsentManageEndpoint.java

```diff
@@ -72,7 +72,7 @@ public ConsentManageEndpoint() {
             initializeConsentManageHandler();
         }
         FinancialServicesConfigParser configParser = FinancialServicesConfigParser.getInstance();
-        allowedHeaderNames = configParser.getServiceExtensionAllowedHeaders();
+        allowedHeaderNames = configParser.getConsentManageExtensionAllowedHeaders();
```

Comment:
with the default config,there's no allowedHeader value. In that case -are we sending the empty value? I feel we have to avoid sending,if the config value is empty/null.

Reply:
Currently we are sending `requestHeaders: {}` if there are no headers to be sent. The other option is to set the value to null in the DTO before sending. In that case `requestHeaders: null` will be sent.

We can avoid sending by changing our serialization method if it is required. But this needs some refactoring.

Reply:
i feel sending that empty/null is not meaningful. Is there a header name/names-which can define as default

Reply:
we can send the client-id as default since it is a mandatory header.